lib/taurus/core/tango/tangoattribute.py

```diff
@@ -600,26 +609,33 @@ def _subscribeEvents(self):
                 self.debug("failed to subscribe to chg events: HW is None")
                 return
 
-        self.__subscription_event = threading.Event()
```

Comment:
I see that you removed the __subscription_event = threading.Event() initialization...
Is this ok? 
where is it initialized now?